src/rvm.ts

```diff
@@ -0,0 +1,147 @@
+const options = [
+  {
+    name: ["--version", "-v"],
+    description: "Show version of rvm",
+  },
+  {
+    name: "--default",
+    description:
+      "When used with ruby selection, sets a default ruby for new shells",
+  },
+  {
+    name: "--debug",
+    description: "Enable debug mode",
+  },
+  {
+    name: "--force",
+    description: "Force install, removes old install & source before install",
+  },
+  {
+    name: "--all",
+    description: "Used with 'rvm list' to display 'most' available versions",
+  },
+  {
+    name: "--summary",
+    description: "Used with 'do' to print out a summary of the commands run",
+  },
+  {
+    name: "C",
+    description:
+      "Custom configure options, comma separated, double quote args that need quoting, default",
+  },
+];
+
+const interpreters = [
+  {
+    name: "rbx",
+    description: "Rubinius",
+    options: [...options],
```

Comment:
Can use shorthand syntax here because it's the same array for each interpreter! (apply to others too)
```suggestion
    options,
```